policy/reviewer/reviewer.go

```diff
@@ -97,6 +94,28 @@ func selectOrgMembers(prctx pull.Context, allOrgs []string, r *rand.Rand) ([]str
 	return orgMembers, nil
 }
 
+func selectAdmins(prctx pull.Context) ([]string, error) {
+	// Resolve all admin teams on the repo, and resolve their user membership
+	var adminUsers []string
+	teams, err := prctx.Teams()
+	if err != nil {
+		return nil, errors.Wrap(err, "Unable to get list of teams collaborators")
```

Comment:
style: almost all other `errors.Wrap` calls in this project use a message like `failed to ...` - can you update these to match?